.vscode/settings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "editor.rulers": [80],
+  "files.exclude": {
+    "**/.git": true,
+    "**/node_modules": true,
+    "**/build": true
+  },
+  "editor.formatOnSave": true,
+  "flow.useNPMPackagedFlow": true,
+  "javascript.validate.enable": false,
+  "prettier.eslintIntegration": true,
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
+    {
+      "language": "typescript",
+      "autoFix": true
+    },
+    {
+      "language": "typescriptreact",
```

Comment:
Should we commit `vscode`? I've never included it and kept the local file

Reply:
I've added it on purpose because of eslint integration issues that we fixed by adding these entries

Reply:
We should. It gives contributors recommendations for plugins, and provides better integration with vscode eslint plugin. We have to configure additional rule for plugin to check typescript files.